src/main/java/bit/javaoop/AbstractManager.java

```diff
@@ -0,0 +1,53 @@
+package bit.javaoop;
+import java.util.List;
+
+public abstract class AbstractManager extends AbstractEmployee {
+
+    private final Float budget = 100_000f;
+    private List <AbstractEmployee> EmployeeList = getEmployeeList();
+
+
+    public AbstractManager(String name, Float salary)
+    {
+       super(name,salary);
+    }
+
+
+    public abstract boolean canHire();
+
+    public boolean isSatisfied()
+    {
+        return getSalary()>10_000.0f;
+    }
+
+
+    public void addEmployee(AbstractEmployee employee)
+    {
+        EmployeeList.add(employee);
+    }
+
+
+    public Float getSalaries() {
+        Float salaries=0f;
+        for(AbstractEmployee e: EmployeeList)
+        {
+            if(e.getPosition().equals("employee")) salaries+=e.getSalary();
+        }
+        return salaries;
+    }
+
+
+    public Integer getAmountOfEmployees()
+    {
+        Integer count=0;
+        for(AbstractEmployee e: EmployeeList)
+        {
+            if(e.getPosition().equals("employee")) count++;
```

Comment:
If a Manager has another Manager as his employee it still is his employee even though it has a manager position. 

So this method should really look like this:

```
public int getEmployeeCount() {
   return employeeList.size();
}
```